src/shared/setup.js

```diff
@@ -146,7 +146,7 @@ const createHandlers = async (handlers) => {
       Logger.info(`Handler Setup - Registering ${JSON.stringify(handler)}!`)
       switch (handler.type) {
         case 'prepare':
-          await RegisterHandlers.transfers.registerPrepareHandlers(handler.fspList)
+          await RegisterHandlers.transfers.registerPrepareHandler()
           if (!Config.HANDLERS_CRON_DISABLED) {
```

Comment:
this condition should be removed as it is no longer applicable as there is a single topic created